istio/datadog_checks/istio/metrics.py

```diff
@@ -194,12 +198,28 @@
     'mixer_config_rule_configs_total': 'config.rule_configs_total',
     'mixer_dispatcher_destinations_per_request': 'dispatcher.destinations_per_request',
     'mixer_dispatcher_instances_per_request': 'dispatcher.instances_per_request',
-    'mixer_handler_daemons_total': 'handler.daemons_total',
+    'mixer_handler_daemons_total': 'handler.daemons',
```

Comment:
Do you need to still change this to `handler.daemons` even though it is already included in your `NON_CONFORMING_METRICS` list?

Reply:
Since the metric is a gauge and not a counter, I figure that it would be better to have a distinction and call the metric istio.mixer.daemons rather than `istio.mixer.daemons_total` 

But I don't feel that deeply about it so I can be leave it as is.